mne/utils/docs.py

```diff
@@ -1389,7 +1390,19 @@
 """
 docdict['subject'] = """
 subject : str
-    The FreeSurfer subject name.
+    The FreeSurfer subject name (will be used to set the FreeSurfer
+    environment variable ``SUBJECT``).
+"""
+docdict['subject_none'] = """
+subject : str | None
+    The FreeSurfer subject name. If ``None``, the value of the environment
+    variable ``SUBJECT`` will be used.
```

Comment:
Most places we allow `subject=None`, it's taken from the class, like in `stc.plot(subject=None)` will use `stc.subject`. It doesn't use the SUBJECT env var I don't think. (I don't think many if any functions respect the SUBJECT env var and I'd rather not start doing so, I think it's going to be error prone.)

Reply:
... so I think what you have below for `stc_subject` is really the only `subject_none` sort of thing we need in practice

Reply:
btw the old docstrings for `subject` for the `plot_stc...`  functions were:

```
        The subject name corresponding to FreeSurfer environment
         variable SUBJECT. If None stc.subject will be used. If that
         is None, the environment will be used.
```

Reply:
… but I just checked `_check_subject()` and indeed it doesn't use the environment variable, so the docstrings were incorrect.